portfolio/src/main/webapp/script.js

```diff
@@ -29,15 +36,58 @@ function loadComments() {
         });
     });
 }
-
+ 
+ 
+function fetchBlobstoreUrlAndShowForm() {
+  fetch('/blobstore-upload')
+      .then((response) => response.text())
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('my-form');
+        messageForm.action = imageUploadUrl;
+      });
+}
+ 
+ 
+function createImageElement(imageUploadUrl) {
+    const liElement = document.createElement('li');
+ 
+    const imgElement = document.createElement('img');
+    imgElement.src = imageUploadUrl;
+ 
+    const messageElement = document.createElement('span');
+    messageElement.innerText = imageUploadUrl.message;
+ 
+    liElement.appendChild(imgElement);
+    liElement.appendChild(messageElement);
+    return commentElement;
+}
+ 
+function createMap() {
+  
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat:  25.784, lng: -80.2267}, zoom: 16});
+ 
+  const trexMarker = new google.maps.Marker({
+    position: {lat: 25.803979, lng: -80.199256},
+    map: map,
+    title: 'Wynwood'
+  });
+ 
+  const trexInfoWindow =
+      new google.maps.InfoWindow({content: 'This is Wynwood one of Miami hottest art district.'});
+  trexInfoWindow.open(map, trexMarker);
+}
+ 
+ 
 function createListElement(comment) {
     const commentElement = document.createElement('li');
     commentElement.className = 'comment';
     commentElement.id = comment.id;
-
+ 
```

Comment:
Nit: I think there are a lot of these little whitespaces that have been accidentally added. If possible, they should be removed (but NOT the new line, just the extra spaces).